backend/BackendMIIT/pom.xml

```diff
@@ -32,7 +32,12 @@
 		<junit.version>5.10.2</junit.version>
 		<mapstruct.version>1.5.5.Final</mapstruct.version>
 		<lombok.version>1.18.30</lombok.version>
+		<pdfbox.version>3.0.2</pdfbox.version>
+		<jsoup.verison>1.17.2</jsoup.verison>
```

Comment:
The actual version is 1.18.1
 Check it [link](https://jsoup.org/download)

Reply:
No, we'll use 1.17.2

Reply:
Why we use not-actual version?